docs/source/how-to/import_new_asset.rst

```diff
@@ -32,27 +33,28 @@ For more details on instanceable assets, please check the Isaac Sim `documentati
 Using URDF Importer
 -------------------
 
-Isaac Sim includes the URDF and MJCF importers by default. These importers support the
-option to import assets as instanceable assets. By selecting this option, the
+For using the URDF importers in the GUI, please check the documentation at `URDF importer`_. For using the URDF importers from Python scripts, we include a utility tool called ``convert_urdf.py``. This script creates an instance of :class:`~sim.converters.UrdfConverterCfg` which
+is then passed to the :class:`~sim.converters.UrdfConverter` class.
+
+The URDF importer has various configuration parameters that can be set to control the behavior of the importer.
+The default values for the importers configuration parameters are specified are in the :class:`~sim.converters.UrdfConverterCfg` class, and they are listed below. We made a few commonly modified settings to be available as command-line arguments when calling the ``convert_urdf.py``, and they are marked with ``*``` in the list. For a comprehensive list of the configuration parameters, please check the the documentation at `URDF importer`_.
+
+Note that when instanceable option is selected, the
 importer will create two USD files: one for all the mesh data and one for
 all the non-mesh data (e.g. joints, rigid bodies, etc.). The prims in the mesh data file are
 referenced in the non-mesh data file. This allows the mesh data (which is often bulky) to be
 loaded into memory only once and used multiple times in a scene.
```

Comment:
This info is now outdated. It seems that there is an instanceable meshes file made but it no longer is used anywhere by the main file (at least for the URDF improter). Instead the meshes are in a different layer inside the main one as "Flattened Prototypes"

Reply:
this is still the case for IsaacSim 4.2. The new way of importing the meshes won't take effect until ISim 4.5.